tests/test_docs_extraction.py

```diff
@@ -18,6 +23,11 @@ class ModelNoDocs(BaseModel):
 
 
 def test_model_docs_extraction():
+
+    # Using a couple dummy decorators to make sure the frame is pointing at
+    # the `class` line:
+    @dec_noop
```

Comment:
I think this would be most useful in `test_model_docs_duplicate_class`, because if decorators are a problem for the frame method then this might only be passing because it's falling back to `inspect`.